core/src/main/java/bio/terra/pearl/core/dao/BaseJdbiDao.java

```diff
@@ -284,7 +284,15 @@ public Optional<T> findByProperty(String columnName, Object columnValue) {
     }
 
 
-    protected List<T> findAllByProperty(String columnName, Object columnValue) {
+    public List<T> findAllByProperty(String columnName, Object columnValue) {
+        if(columnValue == null) {
+            return jdbi.withHandle(handle ->
+                    handle.createQuery("select * from " + tableName + " where " + columnName + " is null;")
+                            .mapTo(clazz)
+                            .list()
+            );
+        }
+
         return jdbi.withHandle(handle ->
                 handle.createQuery("select * from " + tableName + " where " + columnName + " = :columnValue;")
```

Comment:
a bit cleaner...
```suggestion
        String queryString;
        if(columnValue == null) {
            queryString = "select * from %s where %s is null;".formatted(tableName, columnName);
        } else {
            queryString = "select * from %s where %s = :columnValue;".formatted(tableName, columnName);
        }
        return jdbi.withHandle(handle ->
                handle.createQuery(queryString)
```

Reply:
I like this change from a code simplicity perspective, but jdbi throws an exception due to the fact that the bind parameter is unused in the null case, so I'm going to revert it